recipes/stringmlst/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: stringmlst
+  version: "0.3.6.1"
+
+source:
+  url: https://github.com/jordanlab/stringMLST/archive/0.3.6.1.tar.gz
+  md5: 13d8eb71e17abe0294989fef05071776
+
+build:
+  number: 0
+  skip: True  # [osx]
+    
+
+requirements:
+  build:
+    - pyfaidx ==0.4.8.1    
+
+  run:
+    - bwa ==0.7.13
```

Comment:
These versions were indicated that needed to be installed to ensure that the tool works.